### PR TITLE
task7-vote functionality has been added for each article

### PR DESCRIPTION
**Pull request by Hanieh73**
Branch (task7-vote-on-an-article) is done and ready for review and feedback.

### CODE REVIEW

File: nc-news/src/Components/Votes.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import { patchVote } from "../utils";
+
+
+
+const Votes = ({article_id, votes}) => {
+    const [voteNum, setVoteNum] = useState(0)
+const [disabled, setDisabled] = useState(false)
+    const handleVote = (value) => {
+        
+    setDisabled(false)
```
Comment (lewispricey) [inline nc-news/src/Components/Votes.jsx:11]: Is this line of code required?

**Review by lewispricey — CHANGES_REQUESTED** [review]
Great work on this one so far Hanieh, please could you add in error handling for if the request fails (maybe display an error message, unlock the vote button and unoptimicialy render the increased votes)

Once you've added that in please raise another PR :)

File: nc-news/src/Components/Votes.jsx (hunk 1 above)
Comment (Hanieh73) [reply nc-news/src/Components/Votes.jsx]: unnecessary line of code has been removed

File: nc-news/src/utils.js [hunk 2]
```diff
@@ -40,4 +40,15 @@ export const fetchCommentsByArticleId = (article_id) => {
...
 
+}
+
+export const patchVote = (article_id) => {
+    return axios.get(`https://hz-backend-project.onrender.com/api/articles/${article_id}`)
+        .then(({ data }) => {
+     
+return data.votes
+        }).catch((error) => {
+        console.log(error)
```
[the remark this answers is not in the document]
Comment (Hanieh73) [reply nc-news/src/utils.js]: done :)

File: nc-news/src/Components/Votes.jsx (hunk 1 above)
Comment (Hanieh73) [reply nc-news/src/Components/Votes.jsx]: I have added the error msg for votes, but not sure if it's working how it should do!

**Pull request merged**